app/views/admin/configs/show.html.erb

```diff
@@ -1043,6 +1043,12 @@
                 <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:suggested_users][:description] %></div>
               </div>
 
+              <div class="form-group">
```

Comment:
I think we should try to use the crayons classes that we use for checkboxes elsewhere in this view. For example, in the authentication section of this page, we have a `crayons-field--checkbox` class that renders nicely:

```ruby
<fieldset class="config-authentication-form">
  <div class="crayons-field--checkbox">
    <%= f.check_box :prefer_manual_suggested_users, checked: SiteConfig.prefer_manual_suggested_users, class: "crayons-checkbox" %>
    <div>
      <%= admin_config_label :prefer_manual_suggested_users %>
      <p class="crayons-field__description">
        <%= Constants::SiteConfig::DETAILS[:prefer_manual_suggested_users][:description] %>
       </p>
     </div>
   </div>
 </fieldset>
  <br>

  <%= render "form_submission", f: f %>
```

Which renders like this:

<img width="1041" alt="Screen Shot 2020-10-20 at 3 45 38 PM" src="https://user-images.githubusercontent.com/6921610/96652228-5786d200-12eb-11eb-8fd3-dc61d8525f6d.png">

Curious to hear what @lisasy thinks about using `config-authentication-form` here (maybe we ought to rename it if we use it in other places?)

Reply:
I've been using `config-authentication-form` squarely for the authentication section, so I wouldn't use it here. 

@sidemt can you merge master to this? We've made substantial updates to the styling and markup here to reflect more Crayons styling. Getting that merged would be a very cool thing to do.

Reply:
Sounds good; I defer fully to @lisasy on what crayons styles _should_ be used here :)

Reply:
Updated. I referred to the User Experience and Brand > Public checkbox and used the same style.

Reply:
@sidemt the styling looks good to me 👍